src/codelabs/index.md

```diff
@@ -243,9 +243,11 @@ Learn how to use Flutter with other technologies.
   can interact with the text interface or stream a voice
   interaction via the built-in microphone of a mobile device.
 
-* [Building a game with Flutter and Flame][]<br> 
-  Learn how to create a platform game (featuring
-  Dash or Sparky), using the Flame package.
+* [Introduction to Flame with Flutter][]<br> 
+  Build a Breakout clone using the Flame 2D game engine and
+  embedded it in a Flutter wrapper. You will use Flame's Effects
+  to animate and remove components, along with the Google Fonts and
+  Flutter Animate packages to make the whole game look well designed.
```

Comment:
"Flutter Animate" makes it sound part of Flutter itself. Consider using the package names.

```suggestion
  to animate and remove components, along with the `google_fonts` and
  `flutter_animate` packages, to make the whole game look well designed.
```